search-app/project.clj

```diff
@@ -1,17 +1,21 @@
 (defproject nasa-cmr/cmr-search-app "0.1.0-SNAPSHOT"
   :description "Provides a public search API for concepts in the CMR."
   :url "https://github.com/nasa/Common-Metadata-Repository/tree/master/search-app"
-  :exclusions [[cheshire]
-               [clj-time]
-               [com.fasterxml.jackson.core/jackson-core]
-               [commons-codec/commons-codec]
-               [org.apache.httpcomponents/httpclient]
-               [org.clojure/clojure]
-               [org.clojure/tools.reader]
-               [ring/ring-codec]]
+  :exclusions [[org.clojure/clojure]
```

Comment:
sort the exclusions libraries in alphabetic order.

Reply:
Fixed.